src/app/pages/instances/components/all-instances/instance-details/instance-general-info/instance-edit-form/instance-edit-form.component.ts

```diff
@@ -177,6 +186,7 @@ export class InstanceEditFormComponent {
 
     if (this.isVm) {
       payload.secure_boot = values.secure_boot;
+      payload.image_os = values.image_os;
```

Comment:
Please check with Rehan from MW if we need to send `Linux` when user selects `
Use a Linux image (linuxcontainers.org)`.

Reply:
@undsoft - I tried it, and it says that it should be sent only with `ISO` volume.